_posts/2017/2017-12-28-working-at-sdslabs-14.md

```diff
@@ -0,0 +1,45 @@
+I was advised by seniors in ice-breaking sessions to explore various fields and make use of opportunities available here. Initially, programming seemed so diverse that I found it overwhelming. So, I started learning HTML and CSS, as I had somewhat learnt it in class 10. Through this, I was able to get into NSS Technocreative Web-Development cell. Apart from that, I was able to learn Java in my Object-Oriented programming course.
+
+From the beginning, SDSLabs was continuously organising various competitions on Erdős, Codevillage and Backdoor. Through these, I got to know about SDSLabs and became more inquisite in software development. Then, I decided to look into full-stack development during the winter break and get into SDSLabs.
+
+## Entry to SDSLabs
+
+I still remember, it was 2 o'clock on the last day of SDSLabs' interview, and the list of selected members was to be declared by the day end. Selected students were getting called by SDSLabs .On the other hand, I wasn't even called for interview round. <span style="color:grey">(inner me , _"bhai ka fir kat gya"_)</span>.Then I decided to give it one last shot, called a senior and found out that due to some confusion with names, my name was struck off even before getting to interviews. They asked me to come in 5 mins and took my interview, it was conducted in two stages. First round was about general programming and critical thinking, the main motivate of this round was to galvanize me. Next round, was more or less a discussion, and to my enjoyment (and to my relief) they were asking about my point of view on certain problems, and it was really fun.
+
+Two hours later, they announced the final list and my name was there.
+
+## Journey Begins
+
+When I entered SDSLabs, among the first things which grabbed my attention was the pink interior <span style="color: grey">(there’s an interesting story behind that )</span>, which were lined up with achievements of the members and alumni. I was both impressed and inspired by them, and how they were able to achieve so such  within such a short timespan at IITR. At labs, we were expected to be complete beginners and a recap of some basic but important topics was provided. Lecture on various topics ranging from HTML/CSS to networking, and emerging fields like bitcoins and blockchain were held. Through projects we are able to apply it for creating awesome and useful applications. While going through this cycle of learning and testing our knowledge, seniors guided us and helped with our silliest doubts.
+
+During that time in february, Microsoft organised "code.fun.do", my first hackathon, with some help of seniors and spending sleepless nights, a small but full blown application was ready. That encounter with new technologies, developing application and it's delivering was astounding.
```

Comment:
`February` *

Reply:
`... organised "code.fun.do". This was my first hackathon, and with some help of seniors and spending sleepless nights, a small but full blown application was ready.` *